monai/networks/trt_compiler.py

```diff
@@ -217,6 +222,89 @@ def infer(self, stream, use_cuda_graph=False):
         return self.tensors
 
 
+def remove_non_tensors(input_example, remove_constants=True):
+    #
+    # TODO : see if we can instantiate wrappers to handle non-default non-tensors
+    #
+    non_tensors = {}
+    for k, v in input_example.items():
+        if v is None:
+            non_tensors[k] = v
+        elif not torch.is_tensor(v):
+            if remove_constants:
+                non_tensors[k] = v
+            else:
+                input_example[k] = torch.tensor(v)
+
+    for key in non_tensors.keys():
+        # print(f"Removing non-tensor input: {key})")
+        input_example.pop(key)
+    return non_tensors
+
+
+def unroll_input(input_names, input_example):
+    # Simulate list/tuple unrolling during ONNX export
+    unrolled_input = {}
+    for name in input_names:
+        val = input_example[name]
```

Comment:
I think `input_example.get(name, None)` is a better choice here, in case there are any illegal keys.

Thanks,  
Bin

Reply:
Yes we can look more into making this robust for the odd cases.

Reply:
Done.